src/components/Search.tsx

```diff
@@ -1,18 +1,25 @@
 import React from 'react'
-import { View, StyleSheet, TextInput } from 'react-native'
+import { View, StyleSheet, TextInput, Platform } from 'react-native'
 import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
 
 const styles = StyleSheet.create( {
   searchBar: {
-    padding: 10,
+    paddingLeft: Platform.OS === 'android' ? 10 : undefined,
+    padding: Platform.OS === 'ios' ? 10 : undefined,
```

Comment:
>I also don't now why you've defined paddingLeft, followed by padding. Mistake? 

Not a mistake. On android it needs padding on left and on iOS it was to slim so iOS needs padding everywhere. With this both OS' look same.

I like the idea of refactoring it into a consts file.

Reply:
My bad, I misread it (my eyes skimmed over it because I spent more time looking at the `10: undefined`. Why does android need padding on the left and not right? I can experiment later

Reply:
This is the view which wraps the icon and text input. Also in case of Android we will also end up adding a clear button ourselves (my draft PR has it) so this will change again because on iOS we get clear button but not with android :(